sweepbatcher/sweep_batch.go

```diff
@@ -456,6 +462,11 @@ func (b *batch) addSweep(ctx context.Context, sweep *sweep) (bool, error) {
 	b.log.Infof("adding sweep %x", sweep.swapHash[:6])
 	b.sweeps[sweep.swapHash] = *sweep
 
+	// Update FeeRate.
+	if sweep.minFeeRate > b.rbfCache.FeeRate {
```

Comment:
nit: I'd change the order for readability (so: `if b.rbfCache.FeeRate < sweep.minFeeRate`).

Reply:
Thanks! Fixed this and another instance in `sweep_batcher.go`.